controllers/suite/common.go

```diff
@@ -528,20 +528,23 @@ func CreateAndBootstrapCluster(ctx context.Context, k8sClient client.Client, hum
 				Expect(clusterConfig).ToNot(BeNil())
 				Expect(clusterConfig.Config()).ToNot(BeNil())
 
-				cluster, err := humioClient.GetClusters(clusterConfig.Config(), reconcile.Request{NamespacedName: key})
+				client := humioClient.GetHumioHttpClient(clusterConfig.Config(), reconcile.Request{NamespacedName: key})
```

Comment:
There is a package name `client` ("sigs.k8s.io/controller-runtime/pkg/client"), consider renaming this to avoid possible collision issues.